botCommands/debug.js

```diff
@@ -2,8 +2,32 @@ const { registerBotCommand } = require("../botEngine.js");
 
 const command = {
   regex: /(?<!\S)\/debug(?!\S)/,
-  cb: () =>
-    `Looks like this problem can be solved using a debugger: <https://developers.google.com/web/tools/chrome-devtools/javascript>`,
+  cb: ({ content }) => {
+    const query = content.match(/\B\/debug\s?(\S+)?/)[1];
+
+    const template = `Based on the description of your problem, you can get to the root of it using a debugger. Learning how to track down problems like this is an inevitable part of being a developer.`;
+
+    const defaultReply =
+      template +
+      `\n<https://en.wikipedia.org/wiki/Debugging>
+
+To get a helpful resource in javascript or ruby, run \`/debug js\` or \`/debug rb\`.`;
+
+    switch (query) {
+      case "js":
+        return (
+          template +
+          `\n<https://developers.google.com/web/tools/chrome-devtools/javascript>`
+        );
+      case "rb":
+        return (
+          template +
+          `\n<https://www.theodinproject.com/courses/ruby-programming/lessons/debugging#debugging-with-pry-byebug>`
+        );
+      default:
+        return defaultReply;
+    }
+  },
```

Comment:
I'm thinking that we can store these replies in an object instead of using a switch, and then use `js` or `rb` as the key instead of going through a switch statement. From there, tack on the appropriate response which is stored as the value. 

Again, not in love with mixing concatenation and a template string here.